Python/bltinmodule.c

```diff
@@ -578,12 +578,13 @@ format as builtin_format
 
 Return value.__format__(format_spec)
 
-format_spec defaults to the empty string
+format_spec defaults to the empty string.
+See help('FORMATTING') for description.
```

Comment:
I agree with @terryjreedy that the pointer to the other docs isn't quite right. My suggestion would be to replace the second added sentence with:

> See the Format Specification Mini-Language section of help('FORMATTING') for details.